public/client/src/components/RatingsAndRevs/ratingBreakdown.jsx

```diff
@@ -37,11 +37,11 @@ class RatingBreakdown extends React.Component {
         <h3>Ratings and Reviews</h3>
         <Stars average={avgRating} size={25}/>
         <p>{avgRating} Stars, {totalRatings} Ratings</p>
-        <IndividualRatingBreakdown numStars={5} percent={numFiveStars} ratings={ratings[5]}/>
-        <IndividualRatingBreakdown numStars={4} percent={numFourStars} ratings={ratings[4]}/>
-        <IndividualRatingBreakdown numStars={3} percent={numThreeStars} ratings={ratings[3]}/>
-        <IndividualRatingBreakdown numStars={2} percent={numTwoStars} ratings={ratings[2]}/>
-        <IndividualRatingBreakdown numStars={1} percent={numOneStar} ratings={ratings[1]}/>
+        <IndividualRatingBreakdown numStars={5} percent={numFiveStars} ratings={ratings[5]} updateFilters={this.props.updateFilters}/>
+        <IndividualRatingBreakdown numStars={4} percent={numFourStars} ratings={ratings[4]} updateFilters={this.props.updateFilters}/>
+        <IndividualRatingBreakdown numStars={3} percent={numThreeStars} ratings={ratings[3]} updateFilters={this.props.updateFilters}/>
+        <IndividualRatingBreakdown numStars={2} percent={numTwoStars} ratings={ratings[2]} updateFilters={this.props.updateFilters}/>
+        <IndividualRatingBreakdown numStars={1} percent={numOneStar} ratings={ratings[1]} updateFilters={this.props.updateFilters}/>
```

Comment:
potential refactor idea for dryer code : 
create a for loop 
`for (let i=1; i < 6; i++)` 
for each iteration of i, create component : 
`<IndividualRatingBreakdown` numStars={i} percent={`num${i}Stars} ratings={ratings[i]}`

**would need to rename the numOneStar.. variables, but might save some space.